src/app/legacy/containers/Ad/MPU.test.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-import { render, act } from '@testing-library/react';
+import { act } from '@testing-library/react';
 import { StaticRouter } from 'react-router-dom';
 import { RequestContextProvider } from '#contexts/RequestContext';
 import { ToggleContextProvider } from '#contexts/ToggleContext';
 import { FEATURE_INDEX_PAGE } from '#app/routes/utils/pageTypes';
+import { render } from '../../../components/react-testing-library-with-providers';
```

Comment:
I think you can import `act` from our react testing library with providers, as the component has a wildcard export for the original library.

import { render, act } from '../../../components/react-testing-library-with-providers';